Add offlineWarning component (#5)
vtex-apps/pwa-components: docs/OffilineWarning.md

# Promotion Banner

## Description

> `OfflineWarning` is a VTEX Component that is responsible to inform the user when a page couldn't be showed because of missing network connection. This Component can be imported and used by any VTEX App.

Reviewer: ```suggestion
`OfflineWarning` is a VTEX Component that is responsible to inform the user when a page couldn't be displayed due to a network error. This Component can be imported and used by any VTEX App.
```
